private/performOptimizationBobyqa.m

```diff
@@ -0,0 +1,51 @@
+function parameters = performOptimizationBobyqa(parameters, negLogPost, iMS, par0, J_0, options)
+
+    % Definition of index set of optimized parameters
+    freePars = setdiff(1:parameters.number, options.fixedParameters);
```

Comment:
correct me if I am wrong but I thought that everything related to options.fixedParameters is taken care of in getReduced...